Marlin/example_configurations/AlephObjects/TAZ4/Configuration_adv.h

```diff
@@ -901,6 +901,15 @@
                                               // even if it takes longer than DEFAULT_STEPPER_DEACTIVE_TIME.
   //#define PARK_HEAD_ON_PAUSE                // Go to filament change position on pause, return to print position on resume
   //#define HOME_BEFORE_FILAMENT_CHANGE       // Ensure homing has been completed prior to parking for filament change
+
+  //#define FILAMENT_LOAD_UNLOAD_GCODES       // Enable filament loading and unloading directly from prepare menu
+  #if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
```

Comment:
I'm going to revert this commit since `FILAMENT_UNLOAD_RETRACT_LENGTH`, `FILAMENT_UNLOAD_DELAY`, and `FILAMENT_UNLOAD_PURGE_LENGTH` are used as part of the filament change process even if `FILAMENT_LOAD_UNLOAD_GCODES` is not enabled.